backend/src/router.ts

```diff
@@ -33,6 +33,7 @@ router.patch('/pilots/:callsign', pilotController.updatePilot);
 router.patch('/vdgs/:callsign', authMiddleware, pilotController.updatePilot);
 
 router.get('/measures', flowController.getAllMeasures);
+router.patch('/measures/:id', flowController.setMeasureEnable);
```

Comment:
I don't think setMeasureEnable is the correct functionality to be behind this Endpoint. It should be some type of editing the measure and potentially offer more functionality

Reply:
Of couse it shouldn't. I will change that to a more generic "edit measure" function, but for the time being I would only allow to edit the enable flag.